src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```
@@ -112,6 +112,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
+    "Mekong": {
+      "commandName": "Project",
+      "commandLineArgs": "-c mekong --data-dir .data --jsonrpc-enabled true",
```

JSON-RPC is already enabled in the config file.
```suggestion
      "commandLineArgs": "-c mekong --data-dir .data",
```